src/TagsInput/TagsInput.js

```diff
@@ -11,156 +16,154 @@ import InputField from './components/InputField';
 import { StyleSheet } from '../PlatformStyleSheet';
 import { Text } from '../Text';
 
-import { INPUT_MIN_WIDTH, TAGS_MIN_WIDTH } from './constants';
-
 type Props = {|
-  +label: string,
+  +fontSize: number,
   +onChangeText: (value: string) => void,
-  +placeholder: string,
-  +selected?: string[],
+  +tags: string[],
   +disabled?: boolean,
-  +fontSize: number,
+  +label?: string,
   +onClearPress?: () => void,
+  +onKeyPress?: (e: Event) => void,
+  +placeholder?: string,
   +value?: string,
-  +autoFocus: boolean,
```

Comment:
Why did you remove this prop?

Reply:
I will add this props back I don't know.